### ISSUE TITLE
[Exception][WebStorm 2023.2.2] java.util.NoSuchElementException: Collection contains no element matching the predicate.

### ISSUE THREAD

**Issue by aayman997** [Codium-ai/codiumai-jetbrains-release#122]
# User Description
_User did not enter any detailed information._

# Stack Trace
```
java.util.NoSuchElementException: Collection contains no element matching the predicate.
	at ai.codium.language.JsTsLanguageService.getPsiIdentifierElement(JsTsLanguageService.kt:181)
	at ai.codium.ui.CodiumGutterIconLineMarkerProvider.collectSlowLineMarkers(CodiumGutterIconLineMarkerProvider.kt:48)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.queryProviders(LineMarkersPass.java:225)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.lambda$doCollectMarkers$2(LineMarkersPass.java:113)
	at com.intellij.codeInsight.daemon.impl.Divider.divideInsideAndOutsideInOneRoot(Divider.java:95)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.doCollectMarkers(LineMarkersPass.java:103)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.doCollectInformation(LineMarkersPass.java:76)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:57)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$1(PassExecutorService.java:390)
	at com.intellij.platform.diagnostic.telemetry.helpers.TraceKt.runWithSpanIgnoreThrows(trace.kt:77)
	at com.intellij.platform.diagnostic.telemetry.helpers.TraceUtil.runWithSpanThrows(TraceUtil.java:24)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:386)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1133)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$3(PassExecutorService.java:377)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$13(CoreProgressManager.java:604)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:679)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:635)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:603)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:61)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:376)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:352)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:201)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:207)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:350)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:190)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)

```

# Other Information
```
=== About ===
Build version: WebStorm 2023.2.2 Build: #WS-232.9921.42 September 12, 2023
Theme: Dark
JRE: 17.0.8+7-b1000.22, JetBrains s.r.o.
JVM: 17.0.8+7-b1000.22, OpenJDK 64-Bit Server VM, JetBrains s.r.o.
Operating System: Windows 11.0 (amd64)

=== Plugins ===
Custom plugins: [Emoji Support Plugin (3.1.0), SonarLint (9.0.0.75308), Translation (3.5.4), Darkyen's Time Tracker (1.5.3), Grazie Pro (0.3.252), ESLint Restart Service Action (0.0.1), .ignore (4.5.1), Localization (0.0.6), Nest (1.2.21), Svelte (232.9921.36), CodiumAI - Integrity Agent powered by GPT-3.5&4 (0.6.10), Ideolog (222.1.0.0), Unsafe Wilhelm Scream (1.0.2), Grep Console (12.22.0-IJ2021.1), Quokka (1.0.395), String Manipulation (9.11.3), GitToolBox (500.0.20+222), Xcode-Dark Theme (1.2.4), Material Theme UI (8.12.2), Database Tools and SQL for WebStorm (232.9921.42), Code Screenshots (1.8.1), Bootstrap 4, Font awesome 4, Font Awesome 5 Free & Pro live templates/snippets (6.1.0), React Buddy (2023.3-232), Jenkins Control (0.13.18-2023.2), .env files support (2023.2), AI Assistant (232.9826), Atom Material Icons (93.1.0)]
Disabled plugins:[Time Tracking (232.9921.42)]
```

┆Issue is synchronized with this [Trello card](https://trello.com/c/OyRGZTJ9) by [Unito](https://www.unito.io)

**Comment by aldanchenko** [comment]
Hi @aayman997 

Thanks for sharing! We are currently working on a fix. Should be fixed in the next version. I'll let you know when the next version comes out.

Best regards,
Alex

**Comment by aayman997** [comment]
Thanks

**Comment by ArtsiomCh** [comment]
Fixed in 0.6.12
